app/BUILD.bazel

```diff
@@ -348,6 +353,40 @@ android_library(
     visibility = ["//visibility:private"],
 )
 
+# Library for layout resource files
+android_library(
+    name = "databinding_resources",
+    custom_package = "org.oppia.app.databinding",
+    manifest = "src/main/DbAndroidManifest.xml",
+    srcs = BINDING_ADAPTER_FILES,
+    resource_files = glob(["src/main/res/layout*/**"]),
+    enable_data_binding = 1,
+    exports_manifest = True,
+    deps = [
+        ":resources",
```

Comment:
Similar to an earlier comment on a related PR, please sort this deps list.

Reply:
Done